File: courses/cs50/week6/phonebook.py
import csv


with open("phonebook.csv", "a") as file:
    name = input("Name: ")
    number = input("Number: ")

    writer = csv.DictWriter(file, fieldnames=["name", "number"])
    writer.writerow({"name": name, "number": number})

    """
    writer = csv.writer(file)
    writer.writerow([name, number])

    """

"""
file = open("phonebook.csv", "a")

name = input("Name: ")
number = input("Number: ")

writer = csv.writer(file)
writer.writerow([name, number])

file.close()


"""


"""

people = {"David": "+1-0032-0023-0023", "Carter": "+1-322-4123-2333"}

name = input("Name: ")
if name in people:
    number = people[name]
    print(f"Number: {number}")
 
"""

File: courses/cs50/week3/phonebook.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

typedef struct
{
  string name;
  string number;
} person;

int main(void)
{
  person persons[2];
  persons[0].name = "Carter";
  persons[0].number = "+1-111-111-1111";

  persons[1].name = "David";
  persons[1].number = "+1-222-111-2222";
}